<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectMenu</class>
 <widget class="QWidget" name="ConnectMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>284</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>284</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>284</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="ip_input">
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>41</height>
       </size>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>IP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="port_input">
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>41</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>PORT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="connect_btn">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>51</height>
       </size>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="back_btn">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>51</height>
       </size>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
